.github/workflows/tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  unit_tests:
+    runs-on: ubuntu-latest
+    container:
+      image: ccmbenchmark/php:7.1
```

Comment:
Can you set up a matrix using this workflow: https://github.com/marketplace/actions/setup-php-action#matrix-setup ? We should test all live PHP versions (so actually 7.2+), ubuntu should be enough